File: src/androidTest/java/pt/joaomneto/titancompanion/TestCOT.java
package pt.joaomneto.titancompanion;


import android.support.test.filters.LargeTest;
import android.support.test.runner.AndroidJUnit4;

import org.junit.runner.RunWith;

import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook;

import static pt.joaomneto.titancompanion.consts.FightingFantasyGamebook.CITY_OF_THIEVES;

@LargeTest
@RunWith(AndroidJUnit4.class)
public class TestCOT extends TestTFOD{

    @Override
    protected FightingFantasyGamebook getGamebook(){
        return CITY_OF_THIEVES;
    }

}

File: src/androidTest/java/pt/joaomneto/titancompanion/TestCOT.kt
package pt.joaomneto.titancompanion

import androidx.test.ext.junit.runners.AndroidJUnit4
import androidx.test.filters.LargeTest
import org.junit.runner.RunWith
import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook.CITY_OF_THIEVES

@LargeTest
@RunWith(AndroidJUnit4::class)
class TestCOT : TestTFOD() {

    override val gamebook = CITY_OF_THIEVES
}
